### ISSUE TITLE
Enhancement: Calculate the statusbar height to anchor correctly

### ISSUE THREAD

**Issue by bradmartin** [TheOriginalJosh/nativescript-parallax#4]
Here is a function that should be reliable enough to return the android status bar height, which might need to be taken into consideration for setting the top of the anchor position on pre-lollipop devices.

``` JS
 function statHeight(args) {
    var result = 0;
    var resourceId = utils.ad.getApplicationContext().getResources().getIdentifier("status_bar_height", "dimen", "android");
    if (resourceId > 0) {
        result = utils.ad.getApplicationContext().getResources().getDimensionPixelSize(resourceId);
    }
    console.log(result);
    return result;
}
exports.statHeight = statHeight;
```

This is the current demo screen (not same background image 😉 ) but notice the gap between statusbar and the **Anchor** 
![snip20160413_2](https://cloud.githubusercontent.com/assets/6006148/14516725/9b7eb418-01cb-11e6-8494-62a3a1296afe.png)

If I add a button in the **Content** you'll see the **Anchor** covers the gap.

![snip20160413_4](https://cloud.githubusercontent.com/assets/6006148/14516742/b3c6399c-01cb-11e6-9c3d-839d08812fba.png)

**Comment by bradmartin** [comment]
Not entirely sure how the anchor is being set, no time to dig into the code. So take this as a possible improvement to cover some various cases. It might be solved by simply specifying the parallax image be a specific height at minimum.

**Comment by JoshDSommer** [comment]
I've tried duplicating on a couple different devices this with no luck. If you have a chance at some point want to updated to the latest version of the plugin and see it it's still happening?

**Comment by JoshDSommer** [comment]
I think I might know what it its. the inner content isnt' tall enough to push it all the way to the top. I was looking at the gif in your other ticket and it hit me.

**Comment by JoshDSommer** [comment]
I've got a fix in place for this  😄  in verions 1.0.4
